ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -822,6 +834,51 @@ TFLiteCore::cacheInOutTensorPtr ()
   return err;
 }
 
+/**
+ * @brief Internal function to get the option for tf-lite model.
+ */
+static int
+tflite_parseCustomOption (const GstTensorFilterProperties * prop,
+    tflite_option_s * option)
+{
+  if (prop->num_models != 1 || prop->model_files[0] == NULL)
+    return -1;
+
+  option->model_file = prop->model_files[0];
+  option->accelerators = prop->accl_str;
+  option->use_nnapi = FALSE;
+
+  if (prop->custom_properties) {
+    gchar **strv;
+    guint i, len;
+
+    strv = g_strsplit (prop->custom_properties, ",", -1);
+    len = g_strv_length (strv);
+
+    for (i = 0; i < len; ++i) {
+      gchar **pair = g_strsplit (strv[i], ":", -1);
+
+      if (g_strv_length (pair) > 1) {
+        g_strstrip (pair[0]);
+        g_strstrip (pair[1]);
+
+        if (g_ascii_strcasecmp (pair[0], "UseNNAPI") == 0) {
+          if (g_ascii_strcasecmp (pair[1], "true") == 0)
+            option->use_nnapi = TRUE;
```

Comment:
It's be easier for users if  (case incensitive)
"t*", "1", "e*", "on","y*" are all interpreted as true
(true/1/enabled/on/yes)

Reply:
I will consider common function (move _parse_bool_string() in conf.c)